tests/integration_tests/test_hyperopt.py

```diff
@@ -143,6 +146,7 @@ def test_hyperopt_search_alg(
     ray_cluster_7cpu,
     validate_output_feature=False,
     validation_metric=None,
+    split="validation",
```

Comment:
Is this used? Looks like it's overridden by line 181 bwlow.

Reply:
+1 to @tgaddair's comment. It's not being used. Maybe remove line 181?

Reply:
fixed